lindseysatterfield/react-pinterest
# Ls create boards

## Description
Users can now create boards and they are adding to Firebase, along with the user UID.

## Related Issue
#10 

## Motivation and Context
This allows users to create boards. Necessary to eventually add associated pins with a created board.

## How Can This Be Tested?
- Pull down and run `npm start`
- Add a board and check the console to see that your board has been added

## Types of changes
